man/en/sos-upload.1

```diff
@@ -0,0 +1,111 @@
+.TH SOS UPLOAD 1 "July 2024"
+
+.SH NAME
+sos upload \- Upload files like previously generated sos reports or logs to a policy specific location
```

Comment:
`lintian` will bring this as an issue with `bad-whatis-entry` as per PR #3703. So, this ideally needs to be `sos_upload`